prac_05/state_names.py

```diff
@@ -0,0 +1,20 @@
+"""
+CP1404/CP5632 Practical
+State names in a dictionary
+File needs reformatting
+"""
+
+CODE_TO_NAME = {"QLD": "Queensland", "NSW": "New South Wales", "NT": "Northern Territory", "WA": "Western Australia",
+                "ACT": "Australian Capital Territory", "VIC": "Victoria", "TAS": "Tasmania"}
+# print(CODE_TO_NAME)
+
+for code in CODE_TO_NAME:
```

Comment:
Would using ` for code, name in CODE_TO_NAME.items(): ` be more officiant?

Reply:
I don't believe it's more effecient but it's certainly more readable. Changed.

Reply:
I'll retract my statement as running the file through pylint suggested changing it to that as well...